### ISSUE TITLE
May I use a higher version of tensorflow

### ISSUE THREAD

**Issue by xin-peng** [darylperalta/ScanRL#3]
Hello,
Thank you for your excellent work. I am using a new PC whose GPU is only eligible to cuda>=11.2. Thus, I am wondering if I can use tensorflow==2.6.0 instead of 1.4.0. And what else dependence version should I change?

**Comment by darylperalta** [comment]
Yes, theoretically, experiments can be replicated in an environment with updated dependencies.
